homie-controller/src/lib.rs

```diff
@@ -183,7 +182,13 @@ impl HomieController {
                 }
                 Err(HandleError::Fatal(e)) => return Err(e.into()),
                 Ok(event) => return Ok(event),
+            },
+            Incoming::ConnAck(_) => {
+                // We have connected or reconnected, so make our initial subscription to start
+                // discovering Homie devices.
+                self.start().await?;
```

Comment:
Should we notify the user of the library that this has happened as well? Feels like it is more important than it was considered to be previously.

Reply:
Done.